docs/design/lexical_conventions/symbolic_tokens.md

```diff
@@ -0,0 +1,185 @@
+productions, and we may not want to consider `.` to be an operator, because its
+right "operand" is not an expression.
+
+The presence or absence of whitespace around the symbolic token is used to
+determine its fixity, in the same way we expect a human reader to recognize
+them. For example, we want `a* - 4` to treat the `*` as a unary operator and the
+`-` as a binary operator, while `a * -4` treats `*` as a mathematical operation
+and `-` as the negative sign. Hence we can say that the whitespaces plays a
+really important role here, and we use some rules to avoid confusion:
+
+-   There can be no whitespace between a unary operator and its operand.
+-   The whitespace around a binary operator must be consistent: either there is
+    whitespace on both sides or on neither side.
+-   If there is whitespace on neither side of a binary operator, the token
+    before the operator must be an identifier, a literal, or any kind of closing
+    bracket (for example, `)`, `]`, or `}`), and the token after the operator
+    must be an identifier, a literal, or any kind of opening bracket (for
+    example, `(`, `[`, or `{`).
+
+## Details
+
+Symbolic tokens are intended to be used for widely-recognized operators, such as
+the mathematical operators `+`, `*`, `<`, and so on. Those used as operators
+would generally be expected to also be meaningful for some user-defined types,
+and should be candidates for being made overloadable once we support operator
+overloading.
+
+### Symbolic token list
+
+The following is the initial list of symbolic tokens recognized in a Carbon
+source file:
+
+| Token | Explanation                                                                                                |
+| ----- | ---------------------------------------------------------------------------------------------------------- |
+| `*`   | Indirection, multiplication, and forming pointers                                                          |
+| `&`   | Address-of or Bitwise AND                                                                                  |
+| `=`   | Assignment                                                                                                 |
+| `->`  | Return type and `p->x` equivalent to `(*p).x` (in C++)                                                     |
+| `=>`  | Match syntax                                                                                               |
+| `[]`  | Subscript                                                                                                  |
+| `()`  | Function call and function declaration                                                                     |
+| `{}`  | Struct literals, blocks of control flow statements and the bodies of definitions (classes, functions, etc) |
+| `,`   | Separate arguments in a function call, elements of a tuple, or parameters of a function declaration        |
```

Comment:
Also fields of a struct.  As with `()`, we can simplify this to something like "Separate tuple and struct elements", because function argument lists are just tuples.